src/main/java/bio/terra/cda/app/util/NestedColumn.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.cda.app.util;
+
+public class NestedColumn {
+  private final String column;
+  private final String unnestClause;
+
+  public NestedColumn(String column, String unnestClause) {
+    this.column = column;
+    this.unnestClause = unnestClause;
+  }
+
+  public static NestedColumn generate(String qualifiedColumnName) {
+    return Builder.generate(qualifiedColumnName);
+  }
+
+  public static class Builder {
+    /*
+     column -> SELECT DISTINCT(column)
```

Comment:
This is better but still not correctly formatted. In google code style, block comments start with `/*` and end with `*/` and have `* ` on each line.  https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

Reply:
formatted according to style guide.